defaults/main.yml

```diff
@@ -1,13 +1,19 @@
 ---
+platform_client_deploy_flavor: pull-and-build
```

Comment:
I'm rather inclined to make such big behavioral switches explicit. Also thinking that we may have other deployment flavors in the future (i.e. copy to S3) etc...